main.go

```diff
@@ -76,27 +105,73 @@ func cloneorgrepos(ctx context.Context, client *github.Client, org string) error
 	var orgclone sync.WaitGroup
 	//iterating through the repo array
```

Comment:
I think you're missing the `orgclone.Add(1)`?

Reply:
`executeclone` takes a pointer to `orgclone` (`& orgclone ` in the call) and makes the `wg.Add()` call for each thread. After calling `executeclone`, `cloneorgrepos` calls `orgclone.Wait()`